client/src/components/relatedProducts/ProductCard.jsx

```diff
@@ -5,6 +5,22 @@ import helper from '../../helpers.js';
 
 export default function ProductCard(props) {
 
+  const priceElement =
+    <>
+      {props.product.sale ?
+        <div className='card-price'>
+          <div style={{ color: 'red' }}>
+            {props.product.sale}
+          </div>
+          <div style={{ textDecoration: 'line-through', opacity: '50%' }}>
+            {props.product.price}
+          </div>
+        </div> :
+        <div className='card-price'>{props.product.price}</div>
+      }
+    </>
```

Comment:
Created fragment expression to implement conditional styling for sale price and default price. Used as inner function to replace original div element.